src/lib/viewers/doc/Excel.scss

```diff
@@ -0,0 +1,41 @@
+@import '~react-virtualized/styles.css';
+
+.bp-excel-container {
+    background-color: white;
+    height: 100%;
+    width: 100%;
+}
+
+.full-height {
+    height: 100%;
+}
+
+.DragHandle {
+    cursor: row-resize;
+    z-index: 2;
+}
+
+.DragHandleCol {
+    cursor: col-resize;
+    z-index: 2;
+}
+
+/* stylelint-disable */
+.CommentTooltip {
+    background-color: #ffffe1 !important;
+    border-color: #ffffe1 !important;
+    color: #000 !important;
+}
+/* eslint-enable */
+
+.divComment::after {
```

Comment:
is .divComment a CSS specifier on our end?

Reply:
yes